### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by vmaletta**
Please merge

### CODE REVIEW

File: src/main/java/org/ibp/api/brapi/v1/search/SearchResourceBrapi.java [hunk 1]
```diff
@@ -37,4 +38,19 @@
 		return new ResponseEntity<SingleEntityResponse<SearchDto>>(singleGermplasmResponse, HttpStatus.OK);
 
 	}
+
+	@ApiOperation(value = "Post observation units search", notes = "Post observation units search")
+	@RequestMapping(value = "/{crop}/brapi/v1/search/observationunits", method = RequestMethod.POST)
```
Comment (clarysabel) [inline src/main/java/org/ibp/api/brapi/v1/search/SearchResourceBrapi.java]: should this be v2?

File: src/main/java/org/ibp/api/brapi/v2/observation/ObservationUnitResourceBrapi.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	private SearchRequestService searchRequestService;
+
+	@Autowired
+	private StudyService studyService;
+
+	@ApiOperation(value = "Get Observation Unit search", notes = "Get the results of a Observation Unit search request")
+	@RequestMapping(value = "/{crop}/brapi/v2/search/observationunits/{searchResultsDbid}", method = RequestMethod.GET)
+	@ResponseBody
+	@JsonView(PhenotypeSearchDTO.View.PhenotypeBrapiV2.class)
+	public ResponseEntity<EntityListResponse<PhenotypeSearchDTO>> getSearchObservationUnit(
```
Comment (clarysabel) [inline src/main/java/org/ibp/api/brapi/v2/observation/ObservationUnitResourceBrapi.java]: @vmaletta rename to getObservationUnitsSearch

File: src/main/java/org/ibp/api/brapi/v2/observation/ObservationUnitResourceBrapi.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+		try {
+			observationUnitsSearchRequestDto =
+				(ObservationUnitsSearchRequestDto) this.searchRequestService
+					.getSearchRequest(Integer.valueOf(searchResultsDbid), ObservationUnitsSearchRequestDto.class);
+		} catch (final NumberFormatException | MiddlewareException e) {
+			return new ResponseEntity<>(
+				new EntityListResponse<>(new Result<>(new ArrayList<PhenotypeSearchDTO>())).withMessage("no search request found"),
+				HttpStatus.NOT_FOUND);
+		}
+		final PhenotypeSearchRequestDTO phenotypeSearchDTO = new PhenotypeSearchRequestDTO();
```
Comment (clarysabel) [inline src/main/java/org/ibp/api/brapi/v2/observation/ObservationUnitResourceBrapi.java]: @vmaletta I think convenient to create a mapper

File: src/main/java/org/ibp/api/brapi/v2/observation/ObservationUnitResourceBrapi.java (hunk 2 above)
Comment (vmaletta) [reply src/main/java/org/ibp/api/brapi/v2/observation/ObservationUnitResourceBrapi.java]: ok

File: src/main/java/org/ibp/api/brapi/v1/search/SearchResourceBrapi.java (hunk 1 above)
Comment (vmaletta) [reply src/main/java/org/ibp/api/brapi/v1/search/SearchResourceBrapi.java]: done

File: src/main/java/org/ibp/api/brapi/v2/observation/ObservationUnitResourceBrapi.java (hunk 3 above)
Comment (vmaletta) [reply src/main/java/org/ibp/api/brapi/v2/observation/ObservationUnitResourceBrapi.java]: ok

**Pull request merged**